src/main/app-updater.ts

```diff
@@ -17,7 +17,6 @@ function handleAutoUpdateBackChannel(event: Electron.IpcMainEvent, ...[arg]: Upd
     if (arg.now) {
       logger.info(`${AutoUpdateLogPrefix}: User chose to update now`);
       autoUpdater.quitAndInstall(true, true);
-      app.exit(); // this is needed for the installer not to fail on windows.
```

Comment:
Is the whole app exiting without this. I mean that the tray service also is closed?